src/test/java/com/redhat/pantheon/model/ModuleTest.java

```diff
@@ -20,71 +20,71 @@
-                .createModuleLocale(new Locale("es", "ES"))
-                .revisions.getOrCreate()
-                .createModuleRevision("v1");
-        module.getResourceResolver().commit();
-
-        // Then
-        assertNotNull(slingContext.resourceResolver().getResource("/content/module1/locales/es_ES/revisions/v1"));
-    }
-
-    @Test
-    void getRevisionByLocaleAndRevisionName() throws Exception {
-        // Given
-        slingContext.build()
-                // module 1, en_US
-                .resource("/content/module1/locales/en_US/revisions/1")
-                .resource("/content/module1/locales/en_US/revisions/2")
-                // module 1, es_ES
-                .resource("/content/module1/locales/es_ES/revisions/1")
-                .resource("/content/module1/locales/es_ES/revisions/2")
-                // module 2, en_US
-                .resource("/content/module2/locales/en_US/revisions/1")
-                .resource("/content/module2/locales/en_US/revisions/2")
-                // module 2, es_ES
-                .resource("/content/module2/locales/es_ES/revisions/1")
-                .resource("/content/module2/locales/es_ES/revisions/2")
-                .commit();
-        slingContext.registerAdapter(Resource.class, Node.class, mock(Node.class));
-
-        // When
-        Module module1 = new Module(slingContext.resourceResolver().getResource("/content/module1"));
-        Module module2 = new Module(slingContext.resourceResolver().getResource("/content/module2"));
-
-        // Then
-        assertNotNull(
-                module1.locales.get().getModuleLocale(new Locale("en", "US")).revisions.get().getModuleRevision("1"));
-        assertNotNull(
-                module1.locales.get().getModuleLocale(new Locale("en", "US")).revisions.get().getModuleRevision("2"));
-        assertNotNull(
-                module1.locales.get().getModuleLocale(new Locale("es", "ES")).revisions.get().getModuleRevision("1"));
-        assertNotNull(
-                module1.locales.get().getModuleLocale(new Locale("es", "ES")).revisions.get().getModuleRevision("2"));
-        assertNotNull(
-                module2.locales.get().getModuleLocale(new Locale("en", "US")).revisions.get().getModuleRevision("1"));
-        assertNotNull(
-                module2.locales.get().getModuleLocale(new Locale("en", "US")).revisions.get().getModuleRevision("1"));
-        assertNotNull(
-                module2.locales.get().getModuleLocale(new Locale("es", "ES")).revisions.get().getModuleRevision("1"));
-        assertNotNull(
-                module2.locales.get().getModuleLocale(new Locale("es", "ES")).revisions.get().getModuleRevision("2"));
-
-        assertNull(
-                module1.locales.get().getModuleLocale(Locale.SIMPLIFIED_CHINESE));
-        assertNull(
-                module1.locales.get().getModuleLocale(new Locale("es", "ES")).revisions.get().getModuleRevision("abc"));
-    }
+//    @Test
```

Comment:
Any reason we are keeping these instead of deleting these lines?

Reply:
This is a good find, and an oversight. I will uncomment these tests and update.